python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -1032,6 +1062,28 @@ def is_basic_daily(self) -> bool:
     def is_basic_hourly(self) -> bool:
         return self.cron_schedule == "0 * * * *"
 
+    def get_fixed_minute_interval(self) -> Optional[int]:
+        if self.is_basic_hourly:
+            return 60
+
+        cron_parts = self.cron_schedule.split()
+        is_wildcard = [part == "*" for part in cron_parts]
+        if not is_wildcard[1:]:
+            return None
+
+        if not cron_parts[0].startswith("*/"):
+            return None
+
+        try:
+            interval = int(cron_parts[0][2:])
+        except ValueError:
+            return None
+
+        if 60 % interval == 0:
```

Comment:
Should this be `interval % 60`? Not used to seeing the constant on the left side of this operator.